automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/workspaces/WorkspaceSummaryPage.scala

```diff
@@ -186,8 +186,12 @@ class CloneWorkspaceModal(implicit webDriver: WebDriver) extends FireCloudView {
       textField(workspaceNameInput).value = name
     }
 
-    def readPresetAuthDomain(): Option[String] = {
-      find(presetAuthDomain).map(_.text)
+    def readLockedAuthDomainGroups(): List[(String, Boolean)] = {
```

Comment:
`locked-auth-domain-group` is only ever rendered on locked select lists and the name of this function implies that only locked (not enabled) ones are returned, so I think it's redundant to return `List[(String, Boolean)]`. `List[String]` should suffice.

I do like the idea of checking that the elements are actually disabled, though, so some kind of filter is worth keeping. One option would be to put some `data-` attribute on all select lists, renamed this function to `readAuthDomainGroups`, and implement `readLockedAuthDomainGroups` in terms of that.

I tried to pull together a suggested implementation and didn't like any of the results. There's probably an obvious solution that I'm not thinking of, but I can share what I have with you if you're interested.